### DISCUSSION TITLE
Upgrade to docsy v0.9: footer display of 'All rights reserved'

### DISCUSSION THREAD

**Discussion by deining** [google/docsy#1862] [General]
I just upgraded an existing site to newly released docsy v0.9.0. While reading the section [`Footer streamlined`](https://www.docsy.dev/blog/2024/0.9.0/#footer-streamlined) of the new release blog post, I stumbled across two issues:

a) The section gives this fragment for enabling display of `All rights reserved` in the footer:

```
.td-footer__all_rights_reserved {
    display: inline;
}
```

This worked for me only after I added `!important` to the statement.

b) The section tells:

- The All-rights-reserved text is hidden by default. To make it visible, add the
  following to your `_styles_project.scss` project style file, or delete the
  `"footer_all_rights_reserved"` language parameter from all your site's
  internationalization bundles.

For me the use of `or` in this context makes that sound like:

`As an alternative to using a style override, you can also delete the language parameter.`

Is that really true? I don't really see why deleting the language parameters could enable the display of the label.

The only thing I can imagine is something like this:

```
If the text still does not show up after adding the style override,
make sure you didn't disable the text by overriding it via a site
specific language parameter that contains an empty string.
```

@chalin: Is that what you had in mind? Can you please clarify? Thanks!

**Comment by chalin** [comment]
Thanks for the feedback @deining!

(a) `!important` isn't required to enable it for the User Guide, but I'm ok adding a comment that it might be needed (as is done elsewhere).

(b) Oh my, that is a relic from when I was thinking of doing things differently! You are right, of course, that suggested change to i18n files won't add it back 🙃.  I'll remove that phrase.

**Comment by deining** [comment]
Thanks for #1865 which resolves this issue!
